bin-docker/wraith

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+layout=$2
+layout_param=$2
+
+if [[ ! -z "$layout" ]]; then
+  if [ ${layout:0:7} == "wraith/" ]; then
+    layout=${layout:8}
+  else
+    layout_param="wraith/" + $layout
+  fi
+  docker-compose run --rm web bin/rails runner script/before_wraith.rb "$layout"
```

Comment:
does this somehow prevent us from running the next docker-compose?

Reply:
I wouldn't expect it to? but idk how docker

Reply:
I just modified this from the bin-docker script you had there already, iirc, to run the before-wraith stuff.

Reply:
Yeah but my point was 🐛 are you therefore trying to run Docker twice, which seems incorrect. Unfortunately I have no way to test this until new laptop or helpful input from @Throne3d